include/swift/Frontend/FrontendOptions.h

```diff
@@ -62,9 +66,11 @@ enum class InputFileKind {
   IFK_LLVM_IR
 };
 
-/// Options for controlling the behavior of the frontend.
-class FrontendOptions {
-public:
+/// Information about all the inputs to the frontend.
+class FrontendInputs {
+private:
+  // FIXME: (dmu) create a class for the inputs
```

Comment:
Stale comment?

Reply:
Thanks. Removed the comment, but forgot to push before a merge.